### PR TITLE
Ports bay's supermatter monitor modular program

### PR DESCRIPTION
**Pull request by Kmsxkuse**
Closes #1305

### CODE REVIEW

File: code/modules/modular_computers/file_system/programs/sm_monitor.dm.rej [hunk 1]
```diff
@@ -0,0 +1,36 @@
+diff a/code/modules/modular_computers/file_system/programs/sm_monitor.dm b/code/modules/modular_computers/file_system/programs/sm_monitor.dm	(rejected hunks)
+@@ -70,32 +70,27 @@
```
Comment (ninjanomnom) [inline code/modules/modular_computers/file_system/programs/sm_monitor.dm.rej]: you forgot a reject file

**Comment by ike709** [comment]
>not doing mirror PRs in order

s t o p

File: tgui/assets/tgui.css [hunk 2]
```diff
@@ -1 +1 @@
...ata-tooltip].tooltip-bottom:hover:after{-ms-transform:translateX(-50%) translateY(8px);transform:translateX(-50%) translateY(8px)}body.solgov div[data-tooltip].tooltip-left:after,body.solgov span[data-tooltip].tooltip-left:after{top:50%;right:100%;-ms-transform:translateX(8px) translateY(-50%);transform:translateX(8px) translateY(-50%)}body.solgov div[data-tooltip].tooltip-left:hover:after,body.solgov span[data-tooltip].tooltip-left:hover:after{-ms-transform:translateX(-8px) translateY(-50%);transform:translateX(-8px) translateY(-50%)}body.solgov div[data-tooltip].tooltip-right:after,body.solgov span[data-tooltip].tooltip-right:after{top:50%;left:100%;-ms-transform:translateX(-8px) translateY(-50%);transform:translateX(-8px) translateY(-50%)}body.solgov div[data-tooltip].tooltip-right:hover:after,body.solgov span[data-tooltip].tooltip-right:hover:after{-ms-transform:translateX(8px) translateY(-50%);transform:translateX(8px) translateY(-50%)}body.solgov .bar{display:inline-block;position:relative;vertical-align:middle;width:100%;height:20px;line-height:17px;padding:1px;border:1px solid #000;background:#272727}body.solgov .bar .barText{position:absolute;top:0;right:3px}body.solgov .b
```
Comment (ninjanomnom) [inline tgui/assets/tgui.css]: You manualy edited this didn't you. You need to use the build_assets script in tgui after editing src.

File: tgui/assets/tgui.css.rej [hunk 3]
```diff
@@ -0,0 +1,6 @@
+diff a/tgui/assets/tgui.css b/tgui/assets/tgui.css	(rejected hunks)
```
Comment (ninjanomnom) [inline tgui/assets/tgui.css.rej]: another reject

File: tgui/assets/tgui.js.rej [hunk 4]
```diff
@@ -0,0 +1,36 @@
+diff a/tgui/assets/tgui.js b/tgui/assets/tgui.js	(rejected hunks)
```
Comment (ninjanomnom) [inline tgui/assets/tgui.js.rej]: another reject

File: tgui/assets/tgui.css (hunk 2 above)
Comment (ninjanomnom) [inline tgui/assets/tgui.css]: You edited this manualy didn't you. You need to run the build_assets script when you edit anything in the src file. Not edit this yourself.

Comment (Kmsxkuse) [inline tgui/assets/tgui.css]: Oh uhhh.... Im learning this as i go. That makes sense, who would write this code.

**Comment by Kmsxkuse** [comment]
Welp, I'll get around to Build Assets again when we reach this point.

**Comment by Kmsxkuse** [comment]
And now I'm here, one week later.

**Pull request closed without merging**